x11-misc/yabar/yabar-0.4.0_p20190316.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+COMMIT="a0d3fdfed992149b741eb8fcf53f02b5d1a6142e"
+DESCRIPTION="A modern and lightweight status bar for X window managers"
+HOMEPAGE="https://github.com/geommer/yabar"
+SRC_URI="https://github.com/geommer/yabar/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="playerctl"
+
+DEPEND="
+	app-text/asciidoc
+	dev-libs/libconfig:=
+	media-libs/alsa-lib
+	net-wireless/wireless-tools
+	x11-libs/cairo[xcb]
+	x11-libs/gdk-pixbuf:2
+	x11-libs/libxkbcommon[X]
+	x11-libs/pango
+	x11-libs/xcb-util-wm
+	playerctl? ( media-sound/playerctl )
+"
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/${PN}-${COMMIT}"
+
+src_prepare() {
+	default
+	# Respect CFLAGS and LDFLAGS
+	sed -i -e "s:-flto -O2::g" \
+	-e "s:-Wall::" Makefile || die "Failed to update Makefile"
+
+	# Declare VERSION without relying on git,
+	# and replace playerctl dependency option to base name
+	sed -i -e "s/\$(shell git describe)/0.4.0-179-ga0d3fdf/" \
```

Comment:
(already mentioned this in IRC but wanna state it here for the record as well: these 2 sed commands can be combined into one, as sed takes multiple arguments when editing the same file)

Reply:
Fixed